docs/resources/objectstorage/object.md

```diff
@@ -1,6 +1,7 @@
 # oci\_objectstorage\_bucket
 
 Provides an Objectstorage resource for CRUD operations on objects.
+`**Dan: not sure which links to use here**`
```

Comment:
remove

Reply:
In answer to the question: There isn't an obvious choice here, as there is with most resources. I would go with https://docs.us-phoenix-1.oraclecloud.com/api/#/en/objectstorage/20160918/Object/GetObject.